templates/shared/forms/form-fields.html

```diff
@@ -0,0 +1,208 @@
+<section class="p-section">
+  <form {% if isModal %}class="js-modal-form"{% endif %}
+        action="/marketo/submit"
+        method="post"
+        id="mktoForm_{{ formData.formId }}"
+        onsubmit="getCustomFields(event)">
+    {% for fieldset in fieldsets %}
+      <div class="p-section">
+        <hr class="is-fixed-width" />
+        <fieldset class="p-fieldset-section {% if fieldset.id=='ubuntu-versions' %} js-toggle-checkbox-visibility {% elif fieldset.isRequired and fieldset.fields[0].type=='checkbox' %} js-required-checkbox {% endif %}"
```

Comment:
```suggestion
        <fieldset class="p-fieldset-section {% if fieldset.id=='ubuntu-versions' %}js-toggle-checkbox-visibility{% elif fieldset.isRequired and fieldset.fields[0].type=='checkbox' %}js-required-checkbox{% endif %}"
```

Reply:
Don't mind formatting in the files which include forms, it will be removed before merge and those files will be left unchanged